lib/InputOutputItemsUtility.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
```

Comment:
Keep file names in snake case 
No matter what casing you follow for the code 
File names in snake case